src/lib/helpers/handleFileTracking.js

```diff
@@ -22,15 +22,13 @@ const handleFileTracking = ({ functionsPath, publishPath }) => {
     const trackingFile = readFileSync(trackingFilePath, 'utf8')
     const [trackedFunctions, trackedPublish] = trackingFile.split(TRACKING_FILE_SEPARATOR)
     const cleanConfiguredFiles = (trackedFiles, dirPath) => {
-      trackedFiles.forEach((file) => {
-        const filePath = join(dirPath, file.trim('\r'))
```

Comment:
Not sure I understand the original code. `trim` doesn't accept an argument. Also `recursive: true` is not supported for Node.js 10.
Finally I'm not sure why we need `rmdirSync` on Windows since [removeSync](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove-sync.md) does the same without failing on non existent files.

Reply:
<img width="151" alt="Screen Shot 2021-05-31 at 2 47 40 AM" src="https://user-images.githubusercontent.com/57350178/120151748-a3bbbe00-c1ba-11eb-82b4-984f449a986e.png">

basically this work was trial and error on windows when i first did it. it was just to get a bandaid solution out so plugin users could even SORTOF use the CLI, in lieu of a CLI-based solution like having plugins build into a sep dir (instead of changing their src). if all the tests pass still with this diff then thats great!!

Reply:
Sorry I should have been more clear. `trim('\r')` is the same as `trim()` as `trim` doesn't accept an argument.